tests/model/templates/team-accepted-template.model.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { GithubComment } from '../../../src/app/core/models/github/github-comment.model';
+import { TeamAcceptedMessage, TeamAcceptedTemplate } from '../../../src/app/core/models/templates/team-accepted-template.model';
+
+import { TEAM_RESPONSE_MULTIPLE_DISAGREEMENT } from '../../constants/githubcomment.constants';
+
+const EMPTY_BODY_GITHUB_COMMENT = {
+  body: ''
+} as GithubComment;
+
+const ACCEPTED_MESSAGE_GITHUB_COMMENT = {
+  body: TeamAcceptedMessage
+} as GithubComment;
+
+const hasAcceptedComment = [EMPTY_BODY_GITHUB_COMMENT, ACCEPTED_MESSAGE_GITHUB_COMMENT, TEAM_RESPONSE_MULTIPLE_DISAGREEMENT];
```

Comment:
Not sure as to why `TEAM_RESPONSE_MULTIPLE_DISAGREEMENT` is included in the array of comments under `hasAcceptedComment` ! Maybe it was not supposed to be there? :P